Dockerfile

```diff
@@ -16,6 +16,8 @@ RUN DEBIAN_FRONTEND=noninteractive apt update && apt install -y \
     jq \
     && rm -rf /var/lib/apt/lists/*
 
+RUN curl --proto '=https' --tlsv1.2 -sSf https://github.com/mikefarah/yq/releases/download/v4.33.3/yq_linux_amd64 | install -m 755 /dev/stdin /bin/yq && yq --version
```

Comment:
We have to fetch it from GitHub - i.e. we cannot install a recent version with above `apt install`?

```
RUN DEBIAN_FRONTEND=noninteractive apt update && apt install -y \
    curl \
    ca-certificates \
    build-essential \
    pkg-config \
    libssl-dev \
    llvm-dev \
    liblmdb-dev \
    clang \
    cmake \
    jq \
    yq \ <----------------- ?
    && rm -rf /var/lib/apt/lists/*
```

Reply:
It is available on snap, but the snap install does not work well in Docker.